src/pages/Task.tsx

```diff
@@ -197,49 +230,61 @@ const TaskCore: React.FC = () => {
         </PageSider>
         <LogWrapper>
           <PageContent>
-            <StyledTabs selected={selectedTab} setSelected={selectTabHandler}>
-              <Tab name="Logs" id="task-logs-tab">
-                <Logs logLinks={logLinks} />
-              </Tab>
-              <Tab
-                name={
-                  <span>
-                    {failedTestCount ? (
-                      <TabLabelWithBadge
-                        tabLabel="Tests"
-                        badgeVariant="red"
-                        badgeText={failedTestCount}
-                        dataCyBadge="test-tab-badge"
-                      />
-                    ) : (
-                      "Tests"
-                    )}
-                  </span>
-                }
-                id="task-tests-tab"
-              >
-                <TestsTable />
-              </Tab>
-              <Tab
-                name={
-                  <span>
-                    {fileCount !== undefined ? (
-                      <TabLabelWithBadge
-                        tabLabel="Files"
-                        badgeVariant="lightgray"
-                        badgeText={fileCount}
-                        dataCyBadge="files-tab-badge"
-                      />
-                    ) : (
-                      "Files"
-                    )}
-                  </span>
-                }
-                id="task-files-tab"
-              >
-                <FilesTables />
-              </Tab>
-            </StyledTabs>
+            {showBuildBaronTab !== undefined && (
```

Comment:
this condition will always be true since `showBuildBaronTab` will never be undefined

Reply:
discussed over slack